packages/components/.storybook/i18n.js

```diff
@@ -0,0 +1,5 @@
+import { namespaces as tuiNamespaces } from '@talend/locales-tui-components/namespaces';
+import { locales as tuiLocales } from '@talend/locales-tui-components/locales';
+
+export const namespaces = [tuiNamespaces];
+export const locales = [tuiLocales];
```

Comment:
this is a regression from current master which load locales from unpkg and go againts #3692